dub.json

```diff
@@ -29,6 +29,17 @@
             "mainSourceFile": "test.d",
             "targetName": "test",
             "excludedSourceFiles": ["dstep/*", "clang/*"]
+        },
+
+        {
+            "name": "unittest",
+            "mainSourceFile": "test.d",
+            "targetName": "unittest",
+            "sourcePaths": ["dstep", "clang", "unit_tests"],
+            "importPaths": ["dstep"],
+            "lflags-posix": ["-lclang", "-rpath", ".", "-L.", "-L/usr/lib64/llvm", "-L/usr/lib/llvm-3.7/lib"],
+            "lflags-windows": ["+\\", "+clang"],
+            "excludedSourceFiles": ["dstep/driver/*"]
```

Comment:
Right. It's confusing that the command is called `test` and the configuration is called `unittest`.